qa/rpc-tests/test_framework/util.py

```diff
@@ -36,10 +36,15 @@ def sync_blocks(rpc_connections, wait=1):
     """
     Wait until everybody has the same block count
     """
+    t = 0
     while True:
         counts = [ x.getblockcount() for x in rpc_connections ]
+        print "   Syncing...Node Block Counts ==> " + str(counts) + ' Elapsed Time (s): ' + str(t)
         if counts == [ counts[0] ]*len(counts):
             break
+        if t >= 120:
```

Comment:
well, good question..i was just working on the assumption that nodes 
should sync within a few seconds and 120 seconds was far enough out that 
it would be obvious that there was a problem with syncing but I don't 
know what the requirement really is for how long a node should take to 
sync...is there such a requirement?

On 20/08/2015 8:57 AM, Jonas Schnelli wrote:
>
> In qa/rpc-tests/test_framework/util.py 
> <https://github.com/bitcoin/bitcoin/pull/6548#discussion_r37546795>:
>
> >          if counts == [ counts[0] ]*len(counts):
> >              break
> > +        if t >= 120:
>
> Why 120?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/bitcoin/bitcoin/pull/6548/files#r37546795>.
>